utils/githooks/pre-commit.d/30-Jenkinsfile.sh

```diff
@@ -20,6 +20,9 @@ if [ -z "$(_git_diff_cached_files "Jenkinsfile")" ] ; then
 fi
 
 : "${JENKINS_HOST:=build.hpdd.intel.com}"
+echo "Temporary no access to $JENKINS_HOST. Skipping"
```

Comment:
shellcheck is failing. But I think it only complains on files modified so maybe won't be an issue after merge

Reply:
I think shell check is just complaining that some code is now unreachable.

Reply:
Right. But I'm not 100% sure whether shellcheck only runs against modified files or all shell files. So if we merge this, I'm not sure if other PRs will start seeing a failure in shellcheck

Reply:
Unless I'm missing something, it seems it runs against all shell files so would fail on PRs after merge

Reply:
A simple suppression line should fix it.
CodeSpell and Shellcheck are now available as a plugin for Visual Studio Code so we should be catching these things earlier than in PRs.